tests/testapp/settings.py

```diff
@@ -12,10 +12,11 @@
 import os
 from pathlib import Path
 
+from pictures import conf
+
 # Build paths inside the project like this: BASE_DIR / 'subdir'.
 BASE_DIR = Path(__file__).resolve().parent.parent
 
-
```

Comment:
Can you not reformat the file as part of your patch? You are welcome to open another PR to make format changes. I'd like to keep changes concise.

Reply:
Sorry I missed that my editor do that